src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.courses.service;
+
+import nextstep.courses.domain.course.Course;
+import nextstep.courses.domain.course.CourseRepository;
+import nextstep.courses.domain.session.CoverImg;
+import nextstep.courses.domain.session.Period;
+import nextstep.courses.domain.session.Session;
+import nextstep.courses.domain.session.SessionRepository;
+import nextstep.courses.dto.CoverImgDto;
+import nextstep.users.domain.NsUser;
+import nextstep.users.domain.UserRepository;
+
+public class SessionService {
+
+    private final SessionRepository sessionRepository;
+    private final CourseRepository courseRepository;
+    private final UserRepository userRepository;
+
+    public SessionService(SessionRepository sessionRepository, CourseRepository courseRepository, UserRepository userRepository) {
+        this.sessionRepository = sessionRepository;
+        this.courseRepository = courseRepository;
+        this.userRepository = userRepository;
+    }
+
+    // 무료 세션 생성
+    public void createFreeSession(Long courseId, CoverImgDto coverImgDto, Period period) {
+        Course course = courseRepository.findById(courseId);
+        CoverImg coverImg = CoverImg.from(coverImgDto);
```

Comment:
```suggestion
        CoverImg coverImg = coverImgDto.toCoverImg();
```

이런 관점은 어떻게 보시나요?
DTO보다는 도메인이 더 중요한 로직이라고 생각되어서 
DTO가 도메인을 알고 있는 관점으로 생각해보았어요~

Reply:
@wooobo
피드백 감사합니다!!
저는 처음에 도메인 생성에 관한 로직을 정적 팩토리 메서드를 통해 생성하면서 중요한 로직을 처리하는게 좋을거라고 생각했는데 dto에서 생성해주는것도 좋은것 같습니다!!

한가지 질문이 있는데 제가 알기론 DTO에는 기타 로직이 들어가지 않고 순수하게 값만 전달하는 형식의 객체로 사용을 해야한다 라는 얘기를 들었는데, 리뷰어님은 DTO에 위처럼 도메인 생성에 대한 로직을 갖고 있어도 괜찮다고 생각하시는게 맞을까요??

제가 평소에 개발할때는 DTO에 가능하면 메서드를 추가하지 않으려고 하다보니 피드백 해주신 관점을 사실 전혀 생각을 못했어서 추가 질문 남겼습니다!!
감사합니다~~